desktop/core/src/desktop/static/desktop/css/hue.css

```diff
@@ -14,4 +14,4 @@ distributed under the License is distributed on an "AS IS" BASIS,
ample_alt_pagination div.dataTables_info{width:40%}.paging_full_numbers{width:400px;height:22px;line-height:22px}.paging_full_numbers span.paginate_button,.paging_full_numbers span.paginate_active{border:1px solid #989ea4;-webkit-border-radius:5px;-moz-border-radius:5px;padding:2px 5px;margin:0 3px;cursor:pointer;*cursor:hand}.paging_full_numbers span.paginate_button{background-color:#d6d8db}.paging_full_numbers span.paginate_button:hover{background-color:#adb2b6}.paging_full_numbers span.paginate_active{background-color:#51b5fc}table.display tr.even.row_selected td{background-color:#B0BEC5}table.display tr.odd.row_selected td{background-color:#90A4AE}tr.odd.gradeA td.sorting_1{background-color:#D2E6B9}tr.odd.gradeA td.sorting_2{background-color:#D2E6B9}tr.odd.gradeA td.sorting_3{background-color:#D2E6B9}tr.even.gradeA td.sorting_1{background-color:#D2E6B9}tr.even.gradeA td.sorting_2{background-color:#EDF5E2}tr.even.gradeA td.sorting_3{background-color:#EDF5E2}tr.odd.gradeC td.sorting_1{background-color:#d5c9ed}tr.odd.gradeC td.sorting_2{background-color:#d5c9ed}tr.odd.gradeC td.sorting_3{background-color:#d5c9ed}tr.even.gradeC td.sorting_1{background-color:#d5c9ed}tr.even.gradeC td.sorting_2{background-color:#f4f1fa}tr.even.gradeC td.sorting_3{background-color:#f4f1fa}tr.odd.gradeX td.sorting_1{background-color:#fdcfcf}tr.odd.gradeX td.sorting_2{background-color:#fdcfcf}tr.odd.gradeX td.sorting_3{background-color:#fdcfcf}tr.even.gradeX td.sorting_1{background-color:#fdcfcf}tr.even.gradeX td.sorting_2{background-color:#fff2f2}tr.even.gradeX td.sorting_3{background-color:#fff2f2}tr.odd.gradeU td.sorting_1{background-color:#adb2b6}tr.odd.gradeU td.sorting_2{background-color:#adb2b6}tr.odd.gradeU td.sorting_3{background-color:#adb2b6}tr.even.gradeU td.sorting_1{background-color:#d6d8db}tr.even.gradeU td.sorting_2{background-color:#eaebec}tr.even.gradeU td.sorting_3{background-color:#eaebec}.ex_highlight #example tbody tr.even:hover,#example tbody tr.even td.highlighted{background-color:#ffd664}.ex_highlight #example tbody tr.odd:hover,#example tbody tr.odd td.highlighted{background-color:#ffd664}.ex_highlight_row #example tr.even:hover{background-color:#ffd664}.ex_highlight_row #example tr.even:hover td.sorting_1{background-color:#ffd664}.ex_highlight_row #example tr.even:hover td.sorting_2{background-color:#ffd664}.ex_highlight_row #example tr.even:hover td.sorting_3{background-color:#ffd664}.ex_highlight_row #example tr.odd:hover{background-color:#ffd664}.ex_highlight_row #example tr.odd:hover td.sorting_1{background-color:#fdc926}.ex_highlight_row #example tr.odd:hover td.sorting_2{background-color:#ffd664}.ex_highlight_row #example tr.odd:hover td.sorting_3{background-color:#ffd664}table.KeyTable td{border:3px solid transparent}table.KeyTable td.focus{border:3px solid #607D8B}table.display tr.gradeA{background-color:#EDF5E2}table.display tr.gradeC{background-color:#d5c9ed}table.display tr.gradeX{background-color:#fff2f2}table.display tr.gradeU{b
```

Comment:
Was there no change to hue.less?

Reply:
i.e. import of root-wrapped-antd.less somewhere?

Reply:
Nope, root-wrapped-antd.less is already imported into hue.less from an earlier PR https://github.com/cloudera/hue/pull/2996/files#diff-7ddb051492779805fec518b02b60d805888a89b4d4e86114528815dda8dbe21b